web/themes/custom/usagov/templates/field--node--field-wizard-step--wizard-step.html.twig

```diff
@@ -21,7 +21,11 @@
 </div>
 <div id="wizard_border">
 <fieldset class="usa-fieldset maxw-tablet margin-x-auto">
+    {% if element['#object'].title.value == "What type of scam do you need to report?" or element['#object'].title.value == "¿Qué tipo de estafa necesita reportar?" %}
     <legend class="usa-sr-only">{{ element['#object'].body[0].value|raw }}</legend>
+    {% else %}
```

Comment:
I have my doubts here, because you're using "{{ element['#object'].body[0].value|raw }}"  in both the "if" and the "else" clauses -- I suspect one of them is supposed to be different.